scripts/translation2js.yaml

```diff
@@ -1,2 +0,0 @@
-langs: [de,fr,en,it,rm]
-emptyFilename: empty.json
```

Comment:
yes. Basic idea is to only update languages that we want (translations are not always finished at the same time).